tests/TestHelpers/WebDavHelper.php

```diff
@@ -84,35 +84,15 @@ public static function getFileIdForPath(
 		?string $xRequestId = '',
 		?int $davPathVersionToUse = self::DAV_VERSION_NEW
 	): string {
-		$body
-			= '<?xml version="1.0"?>
-				<d:propfind  xmlns:d="DAV:" xmlns:oc="http://owncloud.org/ns">
- 					<d:prop>
-    					<oc:fileid />
-  					</d:prop>
-				</d:propfind>';
-		$response = self::makeDavRequest(
+		return WebDavHelper::getFilePropertyForPath(
```

Comment:
for when we need to get the body for different properties dynamically there `getBodyForPropfind()` which can be used IMO

Reply:
@SwikritiT I forget to mention in the description of this PR I am trying to avoid maximum changes. In follow-up PR I will implement DRY concept related to getFilePropertyForPath. `getFilePropertyForPath` is introduced so that `getFileIdForPath` method remains untouched and we can reuse the same code for etag too.

In my follow-up PR `getFileIdForPath` and `getFileEtagForPath` will be replaced by `getFilePropertyForPath` . Main concept of `getFilePropertyForPath` is to able to use dynamically 
Maybe I can use propfind in place of this 
```
$body
			= "<?xml version='1.0'?>
				<d:propfind  xmlns:d='DAV:' xmlns:oc='http://owncloud.org/ns'>
 					<d:prop>
 					    <$property />
  					</d:prop>
				</d:propfind>";

		$response = self::makeDavRequest(
			$baseUrl,
			$user,
			$password,
			"PROPFIND",
			$path,
			null,
			$xRequestId,
			$body,
			$davPathVersionToUse
		);
```

Reply:
etag should not be required to request preview. so we won't be needing this for now